.docker/bin/dsh

```diff
@@ -186,11 +225,20 @@ is_docker ()
 	return $?
 }
 
```

Comment:
Function below is needed for update. I need to know that Vagrantfile is present and path to it